langkit/documentation.py

```diff
@@ -554,6 +554,39 @@ def instantiate_templates(doc_dict: Dict[str, str]) -> DocDatabase:
         Decrease the reference count for this big integer.
     """,
 
+    #
+    # Iterators
+    #
+
+    'langkit.iterator_type': """
+    % if lang == 'python':
```

Comment:
Please take the habit of not indenting the content inside mako `%if` blocks in the documentation: it can cause problems (as discussed live yesterday).